pkg/analyzer/analyzers/postman/postman.go

```diff
@@ -1,17 +1,114 @@
+//go:generate generate_permissions permissions.yaml permissions.go postman
 package postman
 
 import (
 	"encoding/json"
 	"fmt"
 	"net/http"
 	"os"
+	"strings"
 
 	"github.com/fatih/color"
 	"github.com/jedib0t/go-pretty/table"
 	"github.com/trufflesecurity/trufflehog/v3/pkg/analyzer/analyzers"
 	"github.com/trufflesecurity/trufflehog/v3/pkg/analyzer/config"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/analyzer/pb/analyzerpb"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/context"
 )
 
+var _ analyzers.Analyzer = (*Analyzer)(nil)
+
+type Analyzer struct {
+	Cfg *config.Config
+}
+
+func (Analyzer) Type() analyzerpb.AnalyzerType { return analyzerpb.AnalyzerType_Postman }
+
+func (a Analyzer) Analyze(_ context.Context, credInfo map[string]string) (*analyzers.AnalyzerResult, error) {
+	key, ok := credInfo["key"]
+	if !ok {
+		return nil, fmt.Errorf("missing key in credInfo")
+	}
+	info, err := AnalyzePermissions(a.Cfg, key)
+	if err != nil {
+		return nil, err
+	}
+	return secretInfoToAnalyzerResult(info), nil
+}
+
+func secretInfoToAnalyzerResult(info *SecretInfo) *analyzers.AnalyzerResult {
+	if info == nil {
+		return nil
+	}
+
+	result := analyzers.AnalyzerResult{
+		AnalyzerType:       analyzerpb.AnalyzerType_Postman,
+		Metadata:           nil,
+		UnboundedResources: []analyzers.Resource{},
+	}
+
+	resource := analyzers.Resource{
+		Name:               info.User.User.FullName,
+		FullyQualifiedName: info.User.User.Email,
```

Comment:
I'm not sure if there can be postman tokens across different teams or domains, but do you think it makes sense to add that information to the fully qualified name?

```
<TeamDomain> / <TeamName> / <Email>
```

Reply:
I do not have that knowledge about sharing token across a team. The token I used had empty team domain and name. I can only see the email as unique string.